src/vm/jitinterface.cpp

```diff
@@ -8970,11 +8970,15 @@ CONTRACTL {
         LPCUTF8 namespaceName;
         LPCUTF8 className = methodTable->GetFullyQualifiedNameInfo(&namespaceName);
```

Comment:
Now that these types are all in S.P.Corelib. Do we still need to maintain the code paths for `System.Numerics.dll` and `System.Numerics.Vectors.dll`?

Or, could we remove all the logic around `IsSIMDVectorAssembly`, etc?

Reply:
As a follow-up question, the method here is currently named `isInSIMDModule`, but the name we use in `crossgen2` is `IsSimdType`. Should we also rename all of the `isInSIMDModule` and `...IsInSIMDModule` methods to `isInSIMDType`? (meta: have we determined whether `SIMD` or `Simd` for C++ code? I know its the latter for managed, but we have a mix in native...)

Reply:
> Or, could we remove all the logic around IsSIMDVectorAssembly, etc?

Yes, this can be deleted now.

Reply:
> method here is currently named isInSIMDModule, but the name we use in crossgen2 is IsSimdType

The JIT/EE interface in crossgen2 uses `isInSIMDModule` name. `IsSimdType` is the internal name used by crossgen.

Ideally, this method would deleted from the JIT/EE interface and the JIT would identify these vector types just like any other intrinsic types.

Reply:
>  deleted from the JIT/EE interface 

Or left orphaned...

Reply:
> Or left orphaned...

Is there a preference for deleting vs orphaning? I'm happy to do either, but I believe the former requires revving the JIT interface version, correct?

Reply:
(I'd also opt to do the removal in a follow up PR, after the type forwards are in CoreFX, unless you have a preference otherwise)

Reply:
I guess you can do orphaning. There are a lot of methods that are orphaned and should be deleted.

Example of prior art:

```
IEEMemoryManager* CEEInfo::getMemoryManager()
{
    UNREACHABLE(); // OBSOLETE
    return NULL;
}
```

Reply:
> I'd also opt to do the removal in a follow up PR

Fine with me.

Reply:
I've logged https://github.com/dotnet/coreclr/issues/27402 and assigned it to myself to track this work.